src/Heading/Heading.js

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { headingLevels, headingFonts, headingAlign, headingWeights, headingTracking, headingSizes } from "./Heading.levers";
+import clsxd from 'clsx-dedupe';
+
+/**
+ * Heading Component
+ *
+ */
+export const Heading = ({ className, level = 2, font, weight, tracking, align, size, uppercase, italic, srOnly, children, ref, ...props }) => {
+  // Defaults & Variables.
+  // ---------------------------------------------------------------------------
+  const levers = {};
+  let HeadingTag = '';
+
+  // Levers
+  // ---------------------------------------------------------------------------
+
+  // props.level - allows number or string input
+  if (level && (headingLevels.includes(level) || headingLevels.includes(Number(level)))) {
+    HeadingTag = 'h' + level;
+  }
```

Comment:
Trying the approach to make it work for both number and string input. Are you ok with this? 😬

Reply:
Good enough for me.